contracts/token/MintableToken.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.4.8;
+
+
+import './StandardToken.sol';
+import '../ownership/Ownable.sol';
+
+
+
+/**
+ * Mintable token
+ *
+ * Simple ERC20 Token example, with mintable token creation
+ * Issue:
+ * https://github.com/OpenZeppelin/zeppelin-solidity/issues/120
+ * Based on code by TokenMarketNet:
+ * https://github.com/TokenMarketNet/ico/blob/master/contracts/MintableToken.sol
+ */
+contract MintableToken is StandardToken, Ownable {
+
+  bool public mintingFinished = false;
+  uint public totalSupply = 0;
+
+  modifier canMint() {
+    if(mintingFinished) throw;
+    _;
+  }
+
+  function mint(address _to, uint _amount) onlyOwner canMint returns (bool) {
+    totalSupply = safeAdd(totalSupply, _amount);
+    balances[_to] = safeAdd(balances[_to], _amount);
+    Transfer(0, _to, _amount);
+    return true;
+  }
+
+  function finishMinting() onlyOwner returns (bool) {
+    mintingFinished = true;
+    return true;
+  }
+
+  function recover(address _from) onlyOwner returns (bool) {
```

Comment:
I added in the recover function because I think the owner should be able to destroy tokens from accounts that are lost or stolen.  Would you create a separate ```RecoverableToken``` contract inheriting from ```StandardToken``` or make it into a helper contract?